### ISSUE TITLE
No such file or directory - ./resources/binaries/ffprobe (Errno::ENOENT)

### ISSUE THREAD

**Issue by muranava** [markolson/storyboard#17]
getting this error message:
/Users/mura/.rvm/rubies/ruby-2.1.2/lib/ruby/2.1.0/open3.rb:193:in `spawn': No such file or directory - ./resources/binaries/ffprobe (Errno::ENOENT)

**Comment by liangqi** [comment]
Yes, after https://github.com/liangqi/storyboard/commit/66231e9ebb8a09849f79e7cb7997d33def99bbdd , got same failure... (just a newbie in ruby world)

**Comment by liangqi** [comment]
Change "./resources/binaries/ffprobe" to "ffprobe" in lib/storyboard/extractors/points.rb, should fix the issue.

BTW, perhaps ffmpeg need to be built with --enable-libass option.
